yuniql-core/FileTraceService.cs

```diff
@@ -18,6 +18,9 @@ public FileTraceService()
         ///<inheritdoc/>
         public bool IsDebugEnabled { get; set; } = false;
 
+        ///<inheritdoc/>
+        public bool TraceSensitiveData { get; set; } = false;

```

Comment:
As convention, we make boolean properties starts with `Is` to show clear intent that its true/false. Please name this as `IsTraceSensitiveData` :)